cmd/containeragent/initialize/package_test.go

```diff
@@ -30,23 +30,23 @@ func (*importSuite) TestImports(c *gc.C) {
 		"agent/constants",
 		"api",
 		"api/agent/agent",
-		"api/base",
 		"api/agent/caasapplication",
+		"api/agent/keyupdater",
+		"api/base",
 		"api/common",
 		"api/common/cloudspec",
-		"api/agent/keyupdater",
 		"api/watcher",
 		"apiserver/errors",
-		"rpc/params",
-		"cmd/containeragent/utils",
 		"caas/kubernetes/provider/constants",
-		"cmd/containeragent/utils",
 		"charmhub",
 		"charmhub/path",
 		"charmhub/transport",
 		"cloud",
 		"cmd",
 		"cmd/constants",
+		"cmd/containeragent/utils",
```

Comment:
Twice?

Reply:
I'm innocent. I just sorted it, so now you see :)

```
-		"cmd/containeragent/utils",
		"caas/kubernetes/provider/constants",
-		"cmd/containeragent/utils",
```